packages/devextreme/js/__internal/core/state_manager/preact_signal_value_container_manager.ts

```diff
@@ -0,0 +1,111 @@
+import type * as StateManagementTypes from './types';
+
+function isSignal(value: { brand?: symbol } | unknown):
+  value is StateManagementTypes.ObservableValueContainer {
+  if (value && typeof value === 'object' && 'brand' in value) {
+    return value.brand === Symbol.for('preact-signals');
+  }
+
+  return false;
+}
+
+// eslint-disable-next-line @stylistic/max-len
+export class PreactSignalValueContainerManager implements StateManagementTypes.ValueContainerManager {
```

Comment:
Let's just move `implements` on the next line instead of disabling the ESLint check

```suggestion
export class PreactSignalValueContainerManager
implements StateManagementTypes.ValueContainerManager {
```

Reply:
Changed